crates/biome_js_analyze/src/analyzers/nursery/no_useless_else.rs

```diff
@@ -0,0 +1,192 @@
+use biome_analyze::{
+    context::RuleContext, declare_rule, ActionCategory, Ast, Rule, RuleDiagnostic,
+};
+use biome_console::markup;
+use biome_diagnostics::Applicability;
+use biome_js_factory::make;
+use biome_js_syntax::{AnyJsStatement, JsElseClause, JsIfStatement, JsStatementList};
+use biome_rowan::{AstNode, AstNodeList, BatchMutationExt};
+
+use crate::JsRuleAction;
+
+declare_rule! {
+    /// Disallow `else` block when the `if` block breaks early.
+    ///
+    /// If an `if` block breaks early using a breaking statement (`return`, `break`, `continue`, or `throw`),
+    /// then the `else` block becomes useless.
+    /// Its contents can be placed outside of the block.
+    ///
```

Comment:
should we refer the ESLint [no-else-return](https://eslint.org/docs/latest/rules/no-else-return/)?

Reply:
I don't think so. Although the rule covers `no-elese-return`, there is too much difference?